lib/meadow/data/file_sets.ex

```diff
@@ -99,6 +100,17 @@ defmodule Meadow.Data.FileSets do
     |> Repo.insert()
   end
 
+  def ingest_file_set(attrs \\ %{}) do
+    case create_file_set(attrs) do
+      {:ok, file_set} ->
+        Pipeline.kickoff(file_set, %{role: file_set.role})
+        {:ok, file_set}
+
+      {:error, changeset} ->
+        {:error, changeset}
+    end
+  end
+
```

Comment:
I think we should consider moving this function out of the `FileSets` context. In my understanding, this module is mainly responsible for interacting with the Repo and `FileSet` schema. As another example, the `Works` context doesn't have any references to the `Pipeline` either. Could possibly be moved to `Meadow.Pipeline` or maybe even a new module?